app/models/form_profile.rb

```diff
@@ -184,30 +184,34 @@ def initialize_identity_information(user)
   end
 
   def initialize_contact_information(user)
-    return nil if user.va_profile.nil?
-
+    address = {}
     if user.va_profile&.address
-      address = {
+      address.merge!(
         street: user.va_profile.address.street,
         street2: nil,
         city: user.va_profile.address.city,
         state: user.va_profile.address.state,
         country: user.va_profile.address.country
-      }
+      ).merge!(derive_postal_code(user))
     end
 
-    address.merge!(derive_postal_code(user)) if address.present?
-
-    home_phone = user&.va_profile&.home_phone&.gsub(/[^\d]/, '')
+    pciu_email = extract_pciu_data(user, :pciu_email)
+    pciu_primary_phone = extract_pciu_data(user, :pciu_primary_phone)
 
     FormContactInformation.new(
       address: address,
-      email: user&.email,
-      us_phone: get_us_phone(home_phone),
-      home_phone: home_phone
+      email: pciu_email,
+      us_phone: get_us_phone(pciu_primary_phone),
+      home_phone: pciu_primary_phone
     )
   end
 
+  def extract_pciu_data(user, method)
+    user&.send(method)
+  rescue Common::Exceptions::Forbidden, Common::Exceptions::BackendServiceException
+    return ''
+  end
```

Comment:
Since the point here is to return pre-fill data for forms, I don't want EVSS PCIU service exceptions for retrieving email and phone number to cause an error and prevent ALL form data from being returned.  Written this way, even if the EVSS PCIU service causes an error, the `address` for `FormContactInformation` (which is populated from MVI) can still be populated and returned.

Also, the rescued errors here are logged to Sentry in the EVSS PCIU Service class.

Reply:
can you log the `Common::Exceptions::Forbidden` exceptions? the EVSS PCIU class doesn't seem to be logging those